database/sql/database.sql

```diff
@@ -1,6 +1,6 @@
-DROP DATABASE IF EXISTS `nodel2`;
```

Comment:
@OtavioBernardes there is a reason for that one. After any small/major change in models/structure I just re-install the database from scratch. The project is at the point where i am always tinkering even with the most basic data structures.